dnf/base.py

```diff
@@ -230,6 +230,11 @@ def configure_plugins(self):
         """Run plugins configure() method."""
         self._plugins._run_config()
 
+    def setup_logging_from_conf(self, conf):
```

Comment:
what actually means that `conf`? is it dictionary or it is a file? if it is a file then which format? ini? then which sections?